traffic_ops/v5-client/server_server_capabilities.go

```diff
@@ -52,8 +52,8 @@ func (to *Session) DeleteServerServerCapability(serverID int, serverCapability s
 
 // GetServerServerCapabilities retrieves a list of Server Capabilities that are
 // assigned to Servers.
-func (to *Session) GetServerServerCapabilities(opts RequestOptions) (tc.ServerServerCapabilitiesResponse, toclientlib.ReqInf, error) {
-	var resp tc.ServerServerCapabilitiesResponse
+func (to *Session) GetServerServerCapabilities(opts RequestOptions) (tc.ServerServerCapabilitiesResponseV5, toclientlib.ReqInf, error) {
+	var resp tc.ServerServerCapabilitiesResponseV5
```

Comment:
Suggestion: `GetServerServerCapabilities` should probably be `GetServerServerCapabilitiesV5`

Reply:
We don't want V5 in the struct method name, since this is already the v5 version of the `Session` struct

Reply:
Sounds good